packages/fxa-settings/src/lib/firefox.ts

```diff
@@ -2,7 +2,10 @@ export enum FirefoxCommand {
   AccountDeleted = 'fxaccounts:delete',
   ProfileChanged = 'profile:change',
   PasswordChanged = 'fxaccounts:change_password',
-  FxAStatus = 'fxaccounts:fxa_status',   
+  FxAStatus = 'fxaccounts:fxa_status',
+  FxALogin = 'fxaccounts:login',
+  FxALogout = 'fxaccounts:logout',
+  FxALoaded = 'fxaccounts:loaded',
```

Comment:
nit: Hmm, this is tricky. Why are these last 3 "FxA" prefixed, but not "FxAAccountDeleted" or "FxAPasswordChanged" above on L2+L4? I can almost understand why it's FxAStatus (since the value is `:fxa_status`), but not sure about these last 3 values.